packages/office-ui-fabric-react/src/components/TextField/TextField.tsx

```diff
@@ -72,6 +80,7 @@ export class TextField extends React.Component<ITextFieldProps, ITextFieldState>
     this._delayedValidate = this._async.debounce(this._validate, this.props.deferredValidationTime);
     this._lastValidation = 0;
     this._willMountTriggerValidation = false;
+    this._noOpHandler = () => { /* noop */ };
```

Comment:
instead of having a member that gets initialized with a closure on every instantiation, why not just have a prototype method called _onChange and add the comment inside the method explaining why it is no-op'ing in that. Then it will be defined on the prototype and less work is done overall. Just a minor suggestion.